apps/antalmanac/tests/__snapshots__/download-ics.test.ts.snap

```diff
@@ -50,18 +50,18 @@ Taught by placeholderInstructor1/placeholderInstructor2",
   {
     "end": [
       2024,
-      4,
-      1,
+      8,
+      5,
       3,
       4,
     ],
     "endOutputType": "local",
     "productId": "antalmanac/ics",
-    "recurrenceRule": "FREQ=WEEKLY;BYDAY=MO,WE,FR;INTERVAL=1;COUNT=30",
+    "recurrenceRule": "FREQ=WEEKLY;BYDAY=MO,WE,FR;INTERVAL=1;COUNT=15",
```

Comment:
Huh, do you know why this count changed? It would be good if you could leave a clarifying comment on GitHub before mergin.

Reply:
I'd presume that count refers to the number of events

30 in a standard quarter (10 weeks, 3 classes a week)

15 in a summer session (5 weeks, 3 classes a week)

Reply:
Oh, that makes a lot of sense. Do you think you could re-write the test so it refers to the same term? It would be annoying to have this error periodically just because of adding a new term

Reply:
Sure, I'll take a look in a sec